Bottom: adef94d5e6a0fd668dc4c8079e9e04683548cffd
Top:    5308d094284304161619fdcea2c27347d7349f91
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 16:06:03 +0100

Refresh of hid-gtk-separate-out-board-fli

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b36870d..3db5468 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -81,6 +81,7 @@ typedef struct hid_gc_struct
 }
 hid_gc_struct;
 
+
 static void draw_lead_user (render_priv *priv);
 
 /* Px converts view->pcb, Vx converts pcb->view */
